Automattic/wp-calypso
# Delete editor site launch

Related to #

## Proposed Changes
I couldn't find anywhere this directory was imported, so I'm deleting it (and also removing some unused deps in ETK). Related partially to #74551.

For this feature to load, its PHP would have to be enqueued by something.... and I don't think it is 😁 

I accidentally discovered this today, but when looking back, I found we've discussed this previously: #53118

## Testing Instructions
1. Sync ETK to your sandbox and verify nothing crashes (including checking PHP err logs)
2. See if you can find anywhere this directory is imported.
